plugins/polio/js/src/domains/Campaigns/Evaluations/EvaluationsForms.tsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable camelcase */
+import { TabContext, TabList } from '@mui/lab';
+import { Box, Tab } from '@mui/material';
+import { useSafeIntl } from 'bluesquare-components';
+import { useFormikContext } from 'formik';
+import React, { FunctionComponent, useState } from 'react';
+
+import MESSAGES from '../../../constants/messages';
+
+import { Campaign } from '../../../constants/types';
+import { EvaluationForm } from '../Rounds/EvaluationForm';
+
+export const scopeFormFields = ['separate_scopes_per_round', 'scopes'];
+
+export const EvaluationsForms: FunctionComponent = () => {
+    const { formatMessage } = useSafeIntl();
+    const {
+        values: { rounds = [] },
+    } = useFormikContext<Campaign>();
+    const [currentTab, setCurrentTab] = useState<string>(
+        rounds[0] ? `${rounds[0].number}` : '1',
+    );
+
+    const handleChangeTab = (event, newValue) => {
+        setCurrentTab(newValue);
```

Comment:
There's a `useTabs` hook that handles redirection if you want to deeplink the selected tab in the url